clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java

```diff
@@ -107,8 +107,6 @@ public class TopicConfig {
     public static final String MAX_MESSAGE_BYTES_CONFIG = "max.message.bytes";
     public static final String MAX_MESSAGE_BYTES_DOC =
         "The largest record batch size allowed by Kafka (after compression if compression is enabled). " +
-        "If this is increased and there are consumers older than 0.10.2, the consumers' fetch " +
-        "size must also be increased so that they can fetch record batches this large. " +
         "In the latest message format version, records are always grouped into batches for efficiency. " +
         "In previous message format versions, uncompressed records are not grouped into batches and this " +
         "limit only applies to a single record in that case.";
```

Comment:
We can also delete all these lines. The only thing we need to remain is "The largest record batch size allowed by Kafka (after compression if compression is enabled)."